src/hooks/core/chat/useFetchChannelHook.ts

```diff
@@ -198,6 +199,7 @@ const useFetchChannelHook = () => {
       const isLocationChannel = channel?.channel_type === 2 || channel?.type_channel === 2;
       const isSelfChatChannel = channel?.type === 'messaging' && channel?.members?.length < 2;
       const isCommunityChannel = channel?.type === 'topics';
+      const isCommunityInvitationChannel = channel?.type === 'topicinvitation';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new variable `isCommunityInvitationChannel` is defined but not used in this hunk. Make sure it's used in the subsequent code or consider removing it if it's not necessary.



<!-- This is an auto-generated comment by OSS CodeRabbit -->